context.go

```diff
@@ -31,6 +31,7 @@ const (
 	MIMEPlain             = binding.MIMEPlain
 	MIMEPOSTForm          = binding.MIMEPOSTForm
 	MIMEMultipartPOSTForm = binding.MIMEMultipartPOSTForm
+	BodyBytesKey          = "github.com/gin-gonic/gin/bodyBytes"
```

Comment:
Sorry, I don't understand it's mean, a little surprising.

Reply:
Sure. fixed.

Reply:
I mean: `BodyBytesKey = "BindBodyBytesKey"`, actually I see `github.com/gin-gonic/gin/bodyBytes` at first feel like import, do you think?

Reply:
Ah, yes, but it is a bit dangerous.  `c.Get(key)`'s `key` is only a `string` and this is easy to conflict with user-defined keys.  I meant the full import-pathish strings as a private key.

I grepped sources and found `BindKey = "_gin-gonic/gin/bindkey"` ([here][]).  Following this, I want to use `BodyBytesKey = "_gin-gonic/gin/bodybyteskey"`, how about this?

[here]: https://github.com/delphinus/gin/blob/2c82901ceab6ae53730a3cfcd9839bee11a08f13/utils.go#L17

Reply:
Good, agreed!